docs/system_overview/data_objects.md

```diff
@@ -0,0 +1,343 @@
+ - `coll_id`: Identifier unique to the parent collection associated with the data object that maps to this replica. This column will contain the same value for all replicas of a given data object. The identifiers are defined in `R_COLL_MAIN` which contains all of the information about collections in a given zone.
+ - `data_name`: Logical Name of the data object that maps to this replica. This column will contain the same value for all replicas of a given data object. This value will be unique for all data objects with an identical `coll_id`.
+ - `data_repl_num`: Identifier unique to this replica. This value does not have any special meaning in the context of the iRODS system other than being able to identify a replica independently of its host resource.
+ - `data_version`: This column is no longer used by iRODS.
+ - `data_type_name`: Stores the data type of the data object ('generic', 'tar file', 'gzipFile', 'zipFile', etc.). The list of possible values are listed in the `R_TOKN_MAIN` table where `token_namespace='data_type'`.
+ - `data_size`: Size of the replica in bytes. This may vary between replicas depending on the state of the data and may or may not reflect the state of the data in storage. `ifsck` can be used to detect inconsistencies.
+ - `resc_group_name`: This column is no longer used by iRODS. The value `EMPTY_RESC_GROUP_NAME` is used to discourage legacy queries. 
+ - `resc_name`: This column is no longer used by iRODS. The value `EMPTY_RESC_NAME` is used to discourage legacy queries.  
+ - `data_path`: Physical path of the data in storage for this replica. This is a string which is determined by the storage resource plugin hosting the data. For example, the unixfilesystem resource will specify a slash-delimited fully qualified path to a file on disk in the vault.
+ - `data_owner_name`: The username of the owner of the data object that maps to this replica. This column will contain the same value for all replicas of a given data object. This value is independent from the Access Control List for the data object which allows for multiple owners to exist. The value of this column will never change, even if the user listed is removed from the ACL.
+ - `data_owner_zone`: The zone of the user which owns the data object that maps to this replica. This column will contain the same value for all replicas of a given data object. This will be populated even if the owner is a user in the local zone.
+ - `data_is_dirty`: This is the status of the replica which is represented as an integer from 0 to 4, inclusive. See [Replica Status](#replica-status) for more details about these statuses.
+ - `data_status`: This column is no longer used by iRODS.
+ - `data_checksum`: String representing the checksum of the replica. This may vary between replicas depending on the state of the data and may or may not reflect the state of the data in storage. `ifsck -K` can be used to detect inconsistencies.
```

Comment:
Interesting. I didn't know `ifsck` could be used for consistency checks regarding checksums. I imagine it must do a few more things than `ichksum`?

Reply:
It can also check size, so it really just checks to make sure the thing in the catalog matches what's on the disk. Are you suggesting that we also mention `ichksum` here? Sorry if I missed the point :)

Reply:
No. I was just surprised that `ifsck` was mentioned over `ichksum`.

It's fine as is.

Reply:
Sounds good